### DISCUSSION TITLE
HomeAssistant avviato da portainer

### DISCUSSION THREAD

**Discussion by AndreaFaino** [indomus/forum#1848] [GENERALE - Discussioni generiche]
Salve a tutti vi spiego il mio problema:
avvio homeassistant con comando da terminale su raspberry e tutto funziona bene, ho creato una seconda installazione di homeassistant da portainer ma non so come fare per farle utilizzare la cartella /home/pi/homeassistant già configurata della  della prima.
Gradirei, se possibile, suggerimenti grazie.

**Comment by snakuzzo** [comment]
Se prima non specifichi nel dettaglio che tipo di installazione hai fatto, è impossibile aiutarti

**Reply by AndreaFaino** [reply]
Giusto, le installazioni sono una su docker da comando dal prompt dei comandi del raspberry (che uso regolarmente), la seconda usando portainer.io ed in questa vorrei trasferire la cartella homeassistant della precedente in modo da non dover riconfigurare tutto da capo.

**Reply by snakuzzo** [reply]
Parli di [questa](https://docs.portainer.io/v/ce-2.9/start/install/server/docker/linux) ?

**Reply by AndreaFaino** [reply]
Si esatto.

**Reply by snakuzzo** [reply]
Ok...immagino che tu voglia utilizzare il `bind`.
Quindi, se la directory con la config di HA che risiede sull'host di Core è ad esempio...

```
/myvolumes/homeassistant
```

...ed è quella che vuoi utilizzare come config dir del container, da GUI farai così...

![bbb](https://user-images.githubusercontent.com/7205371/185769564-7e22a8f8-56cf-4510-8daf-1cc62019b801.png)

P.S. ho appena provato e funziona alla perfezione
